lesson08/assignment/spaceland/src/components/TheHeader.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+<div class="TheHeader">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
You can remove this since you're using SCSS! I think this is what is causing the error in the web console